templates/donor-donation-success.html.twig

```diff
@@ -84,3 +84,9 @@
 
   <p>If you have any questions, please contact us.</p>
 {% endblock %}
+{% block cardDetailsHeldByStripe %}
+{% if paymentMethodType is same as('card') %}
+  {# If it was customer_balance then it would not make sense to talk about card details here. #}
+  Your card details are held securely by Stripe. We do not have access to your card details.
```

Comment:
I think ideally we want the HTML block content to still be in a `<p/>`, to guarantee all clients render it as before?

Reply:
Right. Force pushed an amend.